### PR TITLE
Vortex multiplexing streams

### PR DESCRIPTION
**Pull request by reuvenlax**

### CODE REVIEW

**Comment by reuvenlax** [comment]
R: @yirutang

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteUnshardedRecords.java [hunk 1]
```diff
@@ -342,11 +348,13 @@ void flush(RetryManager<AppendRowsResponse, Context<AppendRowsResponse>> retryMa
                   this.currentOffset += inserts.getSerializedRowsCount();
                 }
                 ApiFuture<AppendRowsResponse> response = writeStream.appendRows(offset, protoRows);
-                inflightWaitSecondsDistribution.update(writeStream.getInflightWaitSeconds());
-                if (writeStream.getInflightWaitSeconds() > 5) {
-                  LOG.warn(
-                      "Storage Api write delay more than {} seconds.",
-                      writeStream.getInflightWaitSeconds());
+                if (!usingMultiplexing) {
```
Comment (yirutang) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteUnshardedRecords.java:351]: We are fixing this in this PR:
https://github.com/googleapis/java-bigquerystorage/pull/1835

**Review by yirutang — APPROVED** [review]

**Comment by reuvenlax** [comment]
Run Spotless PreCommit

**Comment by reuvenlax** [comment]
Run SQL_Java11 PreCommit

**Comment by reuvenlax** [comment]
Run Java PreCommit

**Comment by reuvenlax** [comment]
Run Java_PVR_Flink_Docker PreCommit

**Pull request merged**